package.js

```diff
@@ -174,6 +174,7 @@ function packageTask(pkgPath){
 	        	shell.mkdir('-p', tgtPath);
 	        	shell.cp('-R', path.join(dirName, '*'), tgtPath);
 	        	shell.rm(path.join(tgtPath, '*.csproj'));
+	        	shell.rm(path.join(tgtPath, '*.md'));
```

Comment:
@bryanmacfarlane Is this change ok? We are adding markdown files for all our task. We wont want them to show in the build, right?